pkgs/tools/filesystems/kio-admin/default.nix

```diff
@@ -14,4 +14,12 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook ];
   buildInputs = [ qtbase kio ki18n polkit-qt ];
+
+  meta = with lib; {
+    description = "Manage files as administrator using the admin:// KIO protocol.";
```

Comment:
IIRC the contributing guide tells us to not add dots at the end of the description